### PR TITLE
BIA-869: adding extra data and extra test case for 142990_NCMR11_shold_have_HispanicLatinoEthnicity_Zero to catch when a null value un HispanicLatinoEtnicity come in the data.

### PR DESCRIPTION
**Pull request by nsd-daniel**

### CODE REVIEW

File: src/EdFi.AnalyticsMiddleTier.Tests/Dimensions/When_querying_the_StaffSectionDim_view.cs [hunk 1]
```diff
@@ -462,5 +462,22 @@ public void Then_should_have_YearsOfPriorTeachingExperience_Zero()
...
+       : When_querying_the_StaffSectionDim_view
+        {
+            public Given_StaffSection_142990_NCMR11(TestHarnessBase dataStandard) => SetDataStandard(dataStandard);
+
+            private const string _caseIdentifier = "142990_NCMR11";
+
+            [Test]
+            public void Then_should_have_HispanicLatinoEthnicity_Zero()
+            {
+                (bool success, string errorMessage) testResult = DataStandard.RunTestCase<CountResult>($"{TestCasesFolder}.{DataStandard.DataStandardBaseVersion}.{_caseIdentifier}_should_have_HispanicLatinoEthnicity_Zero.xml");
```
Comment (DavidJGapCR) [inline src/EdFi.AnalyticsMiddleTier.Tests/Dimensions/When_querying_the_StaffSectionDim_view.cs:476]: The test will work when you change the result type from `CountResult` to `StaffSectionDim`.
DS 2 still doesn't work. Doesn't work I mean that when I change the second parameter of the COALESCE function to something different to 0, no tests fail.

**Review by DavidJGapCR — CHANGES_REQUESTED** [review]
As the ticket indicates, when I change the HispanicLatinoEthnicity field, so the second value in the COALESCE is something different than 0, a test should fail.

**Review by DavidJGapCR — APPROVED** [review]
